### PR TITLE
Feature update lib

### PR DESCRIPTION
**Pull request by S1rFrancis**
- correct invalid yaml in locales/en.yml
- update ruby version to 2.3.7
- remove lock on activerecord version < 5

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          command: |
+            mkdir /tmp/test-results
+            TEST_FILES="$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)"
+
+            bundle exec rspec --format progress \
+                            --format RspecJunitFormatter \
+                            --out /tmp/test-results/rspec.xml \
+                            --format progress \
+                            $TEST_FILES
+      - run:
```
Comment (RyanMacG) [inline .circleci/config.yml]: Don't need anything codeclimate related

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      - store_test_results:
+          path: /tmp/test-results
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: test-results
+      - store_artifacts:
+          path: coverage
+          prefix: coverage
+
+      - run:
```
Comment (RyanMacG) [inline .circleci/config.yml]: Same here

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+          command: |
+            ./tmp/cc-test-reporter format-coverage -t simplecov -o coverage/codeclimate.json
+
+      # collect reports
+      - store_test_results:
+          path: /tmp/test-results
+      - store_artifacts:
+          path: /tmp/test-results
+          destination: test-results
+      - store_artifacts:
```
Comment (RyanMacG) [inline .circleci/config.yml]: and here

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "Gemfile.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run:
```
Comment (RyanMacG) [inline .circleci/config.yml]: Can remove this since it's for circle CI

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+      # run tests!
+      - run:
+          name: run tests
+          command: |
+            mkdir /tmp/test-results
+            TEST_FILES="$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)"
+
+            bundle exec rspec --format progress \
+                            --format RspecJunitFormatter \
```
Comment (RyanMacG) [inline .circleci/config.yml:53]: You'll need to add this as a dev dependency in the gemspec

Comment (S1rFrancis) [reply .circleci/config.yml:53]: done

**Review by RyanMacG — APPROVED** [review]

**Pull request merged**